CURATOR_PROMPT.md

```
@@ -0,0 +1,321 @@
+`_(Response format simplified for display)_` suffix that
+`parseStructuredResponse` appends when content can't be parsed as
+structured JSON.
+
+Verify in Curator's OpenObserve that no log line contains `[generation
+was interrupted]` for the session.
+
+### 5b. Existing polluted session — self-heals
+
+1. Find a session in Redis that has accumulated `[generation was
+interrupted]` entries from before the fix shipped:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)